src/admin/cms.js

```diff
@@ -136,10 +139,42 @@ Initative.propTypes = {
 	getAsset: PropTypes.object.isRequired
 };
 
+const resourceTags = {resourceTags: globalResourceTags};
+
+const Resources = ({ entry }) => {
+	return <NunjucksPreview
+		entry={entry}
+		path="layouts/resourceDetail.njk"
+		context={({title, focus, source, readability, type, toolPurpose, toolAccessibilityIssues, openSource, link, sharePointUrl, keywords, learnTags, summary, body }) => ({
+			previewMode: true,
+			title,
+			focus,
+			source,
+			readability,
+			type,
+			toolPurpose,
```

Comment:
`toolPurpose`, `toolAccessibilityIssues` and `learnTags` do not show spaces between their list items, in the CMS preview

![image](https://user-images.githubusercontent.com/547933/142130120-f6477f19-1255-4498-b0b9-c41c82aebc2c.png)